app/views/occupation_standards/_similar_results_list.html.erb

```diff
@@ -0,0 +1,59 @@
+<script>
+  function changeVisibility(element_id) {
```

Comment:
We can move the javascript section to use Stimulus. I'm about to sign off for the day, but happy to pair with you tomorrow and show you how to do it. If you feel adventurous, you an take a look at the stimulus controllers we already have

Reply:
I'll check out stimulus tomorrow.